src/include/common/md5.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+/*
+** This code taken from the SQLite test library (can be found at
+** https://www.sqlite.org/sqllogictest/doc/trunk/about.wiki).
+** Originally found on the internet. The original header comment follows this comment.
+** The code has been heavily modified and reformatted
```

Comment:
Better to say `The code has been refactored, but the algorithm stays the same.`?